weblate/html/index.html

```diff
@@ -42,43 +52,55 @@
 </li>
 </ul>
 
+<script>
+$("#views-menu li a").click(function(){
+  
+  $("#views-title").html($(this).text()+' <span class="caret"></span>');
+    
+  });
+</script>
```

Comment:
Please do not use inline javascript, put the code to weblate/static/loader-bootstrap.js